interface/forms/note/new.php

```diff
@@ -38,7 +38,7 @@
             $('.datepicker').datetimepicker({
                 <?php $datetimepicker_timepicker = true; ?>
                 <?php $datetimepicker_showseconds = true; ?>
-                <?php $datetimepicker_formatInput = true; ?>
+                <?php $datetimepicker_formatInput = false; ?>
```

Comment:
@juggernautsei , Just checking that you meant to:
show the time in the datepicker? (this would be new behavior and would need to modify line 87)
show the seconds in the datepicker? (this would new behavior and would need to modify line 87)

After know answers to above, i could let you know which function to use at line 87 to support internationalization of the date formatting (and could then set $datetimepicker_formatInput to true if do that)

Reply:
@bradymiller I have no memory of this post. I don't know what it means.